tests/containers/ReportContainer_spec.js

```diff
@@ -451,4 +451,34 @@ describe('Report Container', () => {
     expect(chartLegend.at(0).props().capitalize).to.be.true;
     expect(chartLegend.at(3).props().capitalize).to.be.false;
   });
+
+  it('Generate test template A4 layout - test auto page break', (done) => {
+    const testTemplate = TemplateProvider.getTestLayoutTemplateWithPageBreaks();
+    const dimensions = { width: 793.70078740155, height: 1122.519685039335 };
```

Comment:
perhaps move  `{ width: 210 * PIXEL_SIZE, height: 297 * PIXEL_SIZE };` to a constant